src/main/resources/assets/iris/lang/de_de.json

```diff
@@ -15,6 +15,8 @@
   "options.iris.apply": "Anwenden",
   "options.iris.refresh": "Aktualisieren",
   "options.iris.openShaderPackFolder": "Shaderpaket-Ordner öffnen ...",
+  "options.iris.shaderPackSettings": "Shaderpaket-Einstellungen...",
+  "options.iris.shaderPackList": "Shaderpaket-Auswahl...",
```

Comment:
Please also include a space here before the `...` at the end. 

I like the idea of an `Auswahl` but it's still not the same as list and the original is a list